### PR TITLE
+flow_parser.0.159.0

### PR DESCRIPTION
**Pull request by alsoijw**
New package version

### CODE REVIEW

File: packages/flow_parser/flow_parser.0.159.0/opam [hunk 1]
```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+name: "flow_parser"
```
Comment (mseri) [inline packages/flow_parser/flow_parser.0.159.0/opam:2]:
```suggestion
```

File: packages/flow_parser/flow_parser.0.159.0/opam [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+version: "0.159.0"
```
Comment (mseri) [inline packages/flow_parser/flow_parser.0.159.0/opam:3]:
```suggestion
```

**Comment by mseri** [comment]
I think you cannot access the `/tmp/` folder from the sandbox. Is this something that changed with opam 2.1 @kit-ty-kate ? Or I recall it wrong?

File: packages/flow_parser/flow_parser.0.159.0/opam [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+maintainer: "flow@fb.com"
+authors: ["Flow Team <flow@fb.com>"]
+homepage: "https://github.com/facebook/flow/tree/master/src/parser"
+bug-reports: "https://github.com/facebook/flow/issues"
+license: "MIT"
+
+build: [ make "-C" "tmp/flow-0.159.0/src/parser" "build-parser" ]
```
Comment (mseri) [inline packages/flow_parser/flow_parser.0.159.0/opam:8]:
```suggestion
build: [ make "-C" "src/parser" "build-parser" ]
```

File: packages/flow_parser/flow_parser.0.159.0/opam [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+install: [ make "-C" "tmp/flow-0.159.0/src/parser" "ocamlfind-install"]
```
Comment (mseri) [inline packages/flow_parser/flow_parser.0.159.0/opam:9]:
```suggestion
install: [ make "-C" "src/parser" "ocamlfind-install"]
```

**Comment by mseri** [comment]
Forget what I said, I think I was completely off-track

**Comment by mseri** [comment]
@kit-ty-kate is this a new warning of ocaml 4.13? Do we need an upper bound on ocaml here?

**Comment by kit-ty-kate** [comment]
> @kit-ty-kate is this a new warning of ocaml 4.13? Do we need an upper bound on ocaml here?

Yes. It's supposed to be a warning by default but this package uses `-warn-error` which is not recommended in releases.

So either the author wants to fix the package or we have to add the `"ocaml" {< "4.13"}` constraint. @alsoijw what do you think?

**Comment by alsoijw** [comment]
@kit-ty-kate, I think it's better to limit the version now.

File: packages/flow_parser/flow_parser.0.159.0/opam [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+bug-reports: "https://github.com/facebook/flow/issues"
+license: "MIT"
+
+build: [ make "-C" "src/parser" "build-parser" ]
+install: [ make "-C" "src/parser" "ocamlfind-install"]
+depends: [
+  "base" {>= "v0.14.1"}
+  "core_kernel" {>= "v0.14.1"}
+  "dtoa" {>= "0.3.2"}
+  "ocaml" {>= "4.10.2"}
```
Comment (mseri) [inline packages/flow_parser/flow_parser.0.159.0/opam:14]:
```suggestion
  "ocaml" {>= "4.10.2" & < "4.13~"}
```

**Comment by mseri** [comment]
Perfect, thanks a lot

**Pull request merged**